python/sbp/client/source.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Mark Fine <mark@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+class Source(object):
```

Comment:
Yeah, good question - I'm not even sure we should have these source/sink classes. Ideally, there would be an interface that each thing requires and then it would consume objects that met those interfaces... I'm going to try and catalog all the things we're doing in SBP now to figure out what we can reduce down to.